docs/UserGuide.md

```diff
@@ -345,22 +321,121 @@ Examples:
 * `c-find John` returns `john` and `John Doe`.
 * `c-find alex david` returns `Alex Yeoh`, `David Li`.<br>
 
-#### 3.7 Locating persons based on matching tags: `c-tag-find`
+#### 3.5 Locating employees based on matching tags: `c-tag-find`
 
-Finds all active(unarchived) contacts that contain the KEYWORD(s) in their tags.
+Finds all employees whose tags contain the one or more `KEYWORD(s)`.
 
-Format: `c-tag-find KEYWORD [MORE_KEYWORDS] ...`
+Format: `c-tag-find KEYWORD [MORE_KEYWORDS]`
 
-* The search is case-insensitive. e.g `Friday` will match `friday`.
-* The order of the keywords does not matter. e.g. `friday monday` will match `monday friday`.
-* Only the tag names are searched.
-* Only full words will be matched e.g. `PartTime` will not match `PartTimes`.
-* Employees whose tag(s) matching at least one keyword will be listed in the `Employee Contact DeatailL` pane
- (i.e. `OR` search).  e.g. `Friday Monday` will return employees whose tags contain `Friday` or `Monday`.
+* The search is case-insensitive. e.g `Friday` will match `friday`.<br>
```

Comment:
Maybe can be more consistent on whether to use e.g. or for example. Is "for example" more formal?

Reply:
Sure, I will use "e.g." for those examples in the brackets, and use "for example, " for those are not. Thank you!